packages/delegation-process/src/model/domain/errors.ts

```diff
@@ -20,6 +22,8 @@ export const errorCodes = {
   operationRestrictedToDelegate: "0010",
   incorrectState: "0011",
   differentEserviceProducer: "0012",
+  delegationContractNotFound: "0013",
+  operationForbidden: "0014",
```

Comment:
We shouldn't need to add `operationForbidden` as a code here since it is already one of the common error codes defined in the `models` package